File: MeusGastos-b-00.01/src/br/com/mltech/modelo/Categoria.java
package br.com.mltech.modelo;

public class Categoria {

	private Long codCategoria;
	private String descricao;

	/**
	 * Construtor vazio
	 */
	public Categoria() {
		super();
		this.codCategoria = null;
		this.descricao = null;
	}

	
	/**
	 * Categoria(Long codCategoria, String descricao) 
	 * 
	 * @param codCategoria
	 * @param descricao
	 */
	public Categoria(Long codCategoria, String descricao) {
		super();
		this.codCategoria = codCategoria;
		this.descricao = descricao;
	}

	/**
	 * 
	 * @return
	 */
	public Long getCodCategoria() {
		return codCategoria;
	}

	/**
	 * 
	 * @param codCategoria
	 */
	public void setCodCategoria(Long codCategoria) {
		this.codCategoria = codCategoria;
	}

	/**
	 * 
	 * @return
	 */
	public String getDescricao() {
		return descricao;
	}

	/**
	 * 
	 * @param descricao
	 */
	public void setDescricao(String descricao) {
		this.descricao = descricao;
	}

	/**
	 * 
	 */
	public String toString() {
		return codCategoria + "-" + descricao;
	}

}

File: despesas2/Categoria.php
<?php
/**
 * 
 * Representa uma categoria onde uma despesa
 * é classificada.
 * 
 * @author maurocl
 *
 */
class Categoria {

  // propriedades

  /**
   *
   * @var int
   */
  private $id;

  /**
   *
   * @var string
   */
  private $descricao;

  // métodos

  /**
   * 
   * 
   * @param $id
   * @param $descricao
   */
  function __construct($id, $descricao) {
    $this->id        = $id;
    $this->descricao = $descricao;
  }

  /**
   *
   * 
   */
  function __destruct() {

  }

  // métodos getters e setters

  /**
   * 
   * Enter description here ...
   */
  function getId() {
    return $this->id;
  }

  /**
   * 
   * Enter description here ...
   */
  function getDescricao() {
    return $this->descricao;
  }

  /**
   * 
   * 
   * @param unknown_type $id
   */
  function setId($id) {
    $this->id = $id;
  }

  /**
   * 
   * 
   * @param unknown_type $descricao
   */
  function setDescricao($descricao) {
    $this->descricao = $descricao;
  }

  /**
   * 
   * 
   */
  function __toString() {
    return "Id: " . $this->getId() . ", Descricao: " . $this->getDescricao();
  }

}

?>